### DISCUSSION TITLE
Toast not positioning correctly

### DISCUSSION THREAD

**Discussion by akzg21** [Blazored/Toast#177] [General]
When I display a toast, it seems the Position parameter has no effect. E.g. I put TopRight and it still shows in the BottomLeft. I have tried to move the component around in the MainLayour.Razor page but it doesn't seem to make a difference.

Also, is there a way to make the toasts stackable? It seems currently when more than one are displayed, it just makes the page bigger and you need to scroll to view the new toasts.

Thanks!
![image](https://user-images.githubusercontent.com/117839328/200859521-d4df1aec-bac7-4e82-98d9-c810044ea839.png)

**Comment by chrissainty** [comment]
Can you provide a screenshot of the toasts as it's hard to understand what you mean by "just makes the page bigger". But looking at your code, you might want to try moving the `BlazoredToasts` component below your `PageTitle` component. If you're having layout issues that could be what is causing it. See the sample apps in the repo as an example.

**Reply by akzg21** [reply]
Thanks for your reply and suggestions. This snip shows an example of making the page bigger. When adding more than one toast, it adds to the bottom of the page as apposed to stacking them. You can see it's added scroll bar.
![image](https://user-images.githubusercontent.com/117839328/201049057-8b1f770f-14ce-416e-84df-656f64ac962d.png)





I tried your suggestion of moving the `BlazoredToasts` Component below the `PageTitle` component and it results in the toasts being displayed almost outside of the app:
![image](https://user-images.githubusercontent.com/117839328/201049597-2744932f-39fa-4429-b936-d24079cbf102.png)

**Reply by chrissainty** [reply]
Looking at you code again, it seems like you've put the Toasts component in a page, rather than in your main layout?

**Reply by akzg21** [reply]
Hey, no I have added the component to the MainLayout.razor
